### PR TITLE
Merge the `release/3.12` branch into `main` post-release

### PR DESCRIPTION
**Pull request by phargogh**
Just a simple merge here post-release!

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 git-lfs
-natcap.invest
+#natcap.invest
+git+https://github.com/natcap/invest.git@release/3.12#egg=natcap.invest
```
Comment (dcdenu4) [inline requirements.txt:3]: Hey @phargogh, I don't think it makes sense to leave this in here right?

**Review by dcdenu4 — COMMENTED** [review]
Hey @phargogh thanks for this. Just one question about the version of `invest` in the requirements file.

File: requirements.txt (hunk 1 above)
Comment (phargogh) [reply requirements.txt]: Ooo good catch!  Yes, I updated it in https://github.com/natcap/invest.users-guide/commit/30464f4c6f73feebd1b8a02b8e0800b9854ce89d to use `main` instead.  We had previously been using the latest released version of InVEST, but I think `main` might be a better fit.

**Comment by phargogh** [comment]
Thanks @dcdenu4 !  I corrected this to build against `main` instead.  Let me know what you think!

**Pull request merged**